### PR TITLE
K8SPS-124 add backoffLimit

### PR DESCRIPTION
**Pull request by nmarukovich**
[K8SPS-124](https://jira.percona.com/browse/K8SPS-124) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

**CHANGE DESCRIPTION**
---
**Problem:**
*Short explanation of the problem.*
Add backoffLimit to control amount of backup retries. 
**Cause:**
*Short explanation of the root cause of the issue if applicable.*

**Solution:**
*Short explanation of the solution we are providing with this PR.*

**CHECKLIST**
---
**Jira**
- [ ] Is the Jira ticket created and referenced properly?
- [ ] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [ ] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [ ] Is an E2E test/test case added for the new feature/change?
- [ ] Are unit tests added where appropriate?

**Config/Logging/Testability**
- [ ] Are all needed new/changed options added to default YAML files?
- [ ] Did we add proper logging messages for operator actions?
- [ ] Did we ensure compatibility with the previous version or cluster upgrade process?
- [ ] Does the change support oldest and newest supported PS version?
- [ ] Does the change support oldest and newest supported Kubernetes version?

[K8SPS-124]: https://perconadev.atlassian.net/browse/K8SPS-124?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by egegunes — DISMISSED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| async-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/async-ignore-annotations.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/auto-config.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/config.log) |
| config-router | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/config-router.log) |
| demand-backup | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/demand-backup.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/gr-demand-backup.log) |
| gr-demand-backup-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/gr-demand-backup-haproxy.log) |
| gr-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/gr-haproxy.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/gr-ignore-annotations.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/gr-init-deploy.log) |
| gr-one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/gr-one-pod.log) |
| gr-recreate | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/gr-recreate.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/gr-scaling.log) |
| gr-self-healing | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/gr-self-healing.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/gr-tls-cert-manager.log) |
| gr-users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/gr-users.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/limits.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/operator-self-healing.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/recreate.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/scaling.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/sidecars.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/smart-update.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/tls-cert-manager.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/df59e4b/version-service.log) |
| We run 30 out of 30|

commit: https://github.com/percona/percona-server-mysql-operator/pull/565/commits/df59e4b15097eb524e170053b1afb9bf4f67e141
image: `perconalab/percona-server-mysql-operator:PR-565-df59e4b`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| async-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/async-ignore-annotations.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/auto-config.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/config.log) |
| config-router | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/config-router.log) |
| demand-backup | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/demand-backup.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/gr-demand-backup.log) |
| gr-demand-backup-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/gr-demand-backup-haproxy.log) |
| gr-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/gr-haproxy.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/gr-ignore-annotations.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/gr-init-deploy.log) |
| gr-one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/gr-one-pod.log) |
| gr-recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/gr-recreate.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/gr-scaling.log) |
| gr-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/gr-self-healing.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/gr-tls-cert-manager.log) |
| gr-users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/gr-users.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/limits.log) |
| monitoring | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/operator-self-healing.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/recreate.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/scaling.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/sidecars.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/smart-update.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/tls-cert-manager.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/5f7b5ec/version-service.log) |
| We run 30 out of 30|

commit: https://github.com/percona/percona-server-mysql-operator/pull/565/commits/5f7b5ecb4ca05fbbfb651281b47d4dd4a55233e6
image: `perconalab/percona-server-mysql-operator:PR-565-5f7b5ec`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| async-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/async-ignore-annotations.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/auto-config.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/config.log) |
| config-router | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/config-router.log) |
| demand-backup | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/demand-backup.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/gr-demand-backup.log) |
| gr-demand-backup-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/gr-demand-backup-haproxy.log) |
| gr-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/gr-haproxy.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/gr-ignore-annotations.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/gr-init-deploy.log) |
| gr-one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/gr-one-pod.log) |
| gr-recreate | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/gr-recreate.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/gr-scaling.log) |
| gr-self-healing | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/gr-self-healing.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/gr-tls-cert-manager.log) |
| gr-users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/gr-users.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/limits.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/operator-self-healing.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/recreate.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/scaling.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/sidecars.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/smart-update.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/tls-cert-manager.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/66c8f62/version-service.log) |
| We run 30 out of 30|

commit: https://github.com/percona/percona-server-mysql-operator/pull/565/commits/66c8f627888f6ca5076ac0816272243af4199787
image: `perconalab/percona-server-mysql-operator:PR-565-66c8f62`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| async-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/async-ignore-annotations.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/auto-config.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/config.log) |
| config-router | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/config-router.log) |
| demand-backup | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/demand-backup.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/gr-demand-backup.log) |
| gr-demand-backup-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/gr-demand-backup-haproxy.log) |
| gr-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/gr-haproxy.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/gr-ignore-annotations.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/gr-init-deploy.log) |
| gr-one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/gr-one-pod.log) |
| gr-recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/gr-recreate.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/gr-scaling.log) |
| gr-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/gr-self-healing.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/gr-tls-cert-manager.log) |
| gr-users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/gr-users.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/limits.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/operator-self-healing.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/recreate.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/scaling.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/sidecars.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/smart-update.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/tls-cert-manager.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/d52584e/version-service.log) |
| We run 30 out of 30|

commit: https://github.com/percona/percona-server-mysql-operator/pull/565/commits/d52584e116971c1c5d30c03fbaa9faebc464945e
image: `perconalab/percona-server-mysql-operator:PR-565-d52584e`

File: e2e-tests/tests/gr-demand-backup/02-assert.yaml [hunk 1]
```diff
@@ -28,6 +28,10 @@ apiVersion: ps.percona.com/v1alpha1
 kind: PerconaServerMySQL
 metadata:
   name: gr-demand-backup
+spec:
+    allowUnsafeConfigurations: false
```
Comment (egegunes) [inline e2e-tests/tests/gr-demand-backup/02-assert.yaml:32]: indentation seems off here

**Review by inelpandzic — DISMISSED** [review]

**Review by hors — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| async-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/async-ignore-annotations.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/auto-config.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/config.log) |
| config-router | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/config-router.log) |
| demand-backup | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/demand-backup.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/gr-demand-backup.log) |
| gr-demand-backup-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/gr-demand-backup-haproxy.log) |
| gr-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/gr-haproxy.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/gr-ignore-annotations.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/gr-init-deploy.log) |
| gr-one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/gr-one-pod.log) |
| gr-recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/gr-recreate.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/gr-scaling.log) |
| gr-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/gr-self-healing.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/gr-tls-cert-manager.log) |
| gr-users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/gr-users.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/limits.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/operator-self-healing.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/recreate.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/scaling.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/sidecars.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/smart-update.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/tls-cert-manager.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3abe2b2/version-service.log) |
| We run 30 out of 30|

commit: https://github.com/percona/percona-server-mysql-operator/pull/565/commits/3abe2b2d0632b26d2cfa34af361ecfce14e09e77
image: `perconalab/percona-server-mysql-operator:PR-565-3abe2b2`

**Review by egegunes — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| async-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/async-ignore-annotations.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/auto-config.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/config.log) |
| config-router | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/config-router.log) |
| demand-backup | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/demand-backup.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/gr-demand-backup.log) |
| gr-demand-backup-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/gr-demand-backup-haproxy.log) |
| gr-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/gr-haproxy.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/gr-ignore-annotations.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/gr-init-deploy.log) |
| gr-one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/gr-one-pod.log) |
| gr-recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/gr-recreate.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/gr-scaling.log) |
| gr-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/gr-self-healing.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/gr-tls-cert-manager.log) |
| gr-users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/gr-users.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/limits.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/operator-self-healing.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/recreate.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/scaling.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/sidecars.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/smart-update.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/tls-cert-manager.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-565/3a44982/version-service.log) |
| We run 30 out of 30|

commit: https://github.com/percona/percona-server-mysql-operator/pull/565/commits/3a44982545ee4f5ddf47019d757a1d4b7703e4c7
image: `perconalab/percona-server-mysql-operator:PR-565-3a44982`

**Review by hors — APPROVED** [review]

**Pull request merged**